src/plugins/multimedia/assets/mediacontrols.html

```diff
@@ -9,7 +9,8 @@
 		<div class="btn-group">
 			<button type="button" class="btn btn-default playpause" aria-controls="{{mId}}"  aria-labelledby="{{mId}}" title="{{play}}" data-state-on="{{play}}" data-state-off="{{pause}}"><span class="glyphicon glyphicon-play"><span class="wb-inv">{{play}}</span></span></button>
 			<button type="button" class="btn btn-default mute" aria-controls="{{mId}}" aria-labelledby="{{mId}}" title="{{mute_on}}" data-state-on="{{mute_on}}" data-state-off="{{mute_off}}" aria-pressed="false"><span class="glyphicon glyphicon-volume-up"><span class="wb-inv">{{mute_on}}</span></span></button>
-			<input type="range" class="volume" aria-controls="{{mId}}" aria-labelledby="{{mId}}" title="{{volume}}" min="0" max="100" value="100" step="5" />
+			<span id="{{mId}}-vlm-lbl" hidden>{{volume}} 1</span>
```

Comment:
> The `span` element doesn't mean anything on its own

ref. https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-span-element

Reply:
What about the [HTML spec's definition of paragraphs](https://html.spec.whatwg.org/multipage/dom.html#paragraphs)?

The ``p`` element is not the only means by which a paragraph can be represented (although it's obviously the most explicit way of denoting it).

From my understanding, what's currently being done in mediacontrols.html (**without** my PR's changes) technically represents an "implied" paragraph of buttons and inputs (since they're adjacent elements whose content model is [phrasing content](https://html.spec.whatwg.org/multipage/dom.html#phrasing-content-2)).

Using using a ``span`` element to label the input would fit in with that structure since it's also phrasing content. By comparison, using a ``p`` element would technically represent 3 paragraphs (since the "real" paragraph in-between the "implied" ones would break the sequence of adjacent phrasing elements).

So semantically, IMO the way I'm proposing (``span``) would be more semantically-correct than the way you're proposing (``p``). But I don't mind amending this PR to use ``p`` if you insist. Either way is better than the status quo in any case :).

Reply:
The ask here is to be explicit and not relying on implicit semantic.

Reply:
I still personally disagree with using ``p`` over ``span`` in this scenario since the group of buttons/inputs were already an implicit paragraph. My intent was to modify something inside of it - not introduce a new paragraph.

Anyway, I've updated this PR to use ``p`` to bring this PR back to life :).